packages/material-ui/src/Button/Button.js

```diff
@@ -16,7 +16,7 @@ export const styles = theme => ({
     minWidth: theme.spacing.unit * 11,
     minHeight: 36,
     padding: `${theme.spacing.unit}px ${theme.spacing.unit * 2}px`,
-    borderRadius: 2,
```

Comment:
Maybe this should become a property of the theme?

Reply:
The default border radius hasn't changed. https://github.com/mui-org/material-ui/pull/11497#issue-189205439

(But putting it in the theme is a logical next step for v2.)